k8s-manifests/plugin/dsa/config_dsa/setup_dsa.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
```

Comment:
Please rebase the PR to latest main, and pass the shell check.

Reply:
Done.